src/irc/core/irc-servers.c

```diff
@@ -201,6 +202,10 @@ static void server_init(IRC_SERVER_REC *server)
 
 	conn = server->connrec;
 
+	// server->cap_queue = g_slist_append(server->cap_active, "identify-msg");
+	server->cap_queue = g_slist_append(server->cap_active, "sasl");
```

Comment:
`s/cap_active/cap_queue/`

Also I think `g_strdup()`-ing the values might be saner.

Also there should be a way to allow scripts to request caps before registration is finished